services/api/service.go

```diff
@@ -2114,11 +2118,25 @@ func (api *RelayAPI) updateRedisBid(opts redisUpdateBidOpts) (*datastore.SaveBid
 	return &updateBidResult, getPayloadResponse, true
 }
 
+func (api *RelayAPI) getForkFromSlot(slot uint64) spec.DataVersion {
+	if api.isFulu(slot) {
+		return spec.DataVersionFulu
+	} else if api.isElectra(slot) {
+		return spec.DataVersionElectra
+	} else if api.isDeneb(slot) {
+		return spec.DataVersionDeneb
+	} else if api.isCapella(slot) {
+		return spec.DataVersionCapella
+	}
+	return spec.DataVersionPhase0
+}
```

Comment:
This would be a little cleaner as a switch:

```go
func (api *RelayAPI) getForkFromSlot(slot uint64) spec.DataVersion {
	switch {
	case api.isFulu(slot):
		return spec.DataVersionFulu
	case api.isElectra(slot):
		return spec.DataVersionElectra
	case api.isDeneb(slot):
		return spec.DataVersionDeneb
	case api.isCapella(slot):
		return spec.DataVersionCapella
	default:
		return spec.DataVersionPhase0
	}
}
```